3d_segmentation/brats_segmentation_3d.ipynb

```diff
@@ -47,7 +47,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        download=False,
</pre><p> </p><p>Please don't change to False.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>